### ISSUE TITLE
Study New Language Challenge with Duolingo app

### ISSUE THREAD

**Issue by hanzoh1** [wearekickback/challenges#3]
## Study New Language Challenge with Duolingo app

Have you ever tried to learn new language but gave up doing ? 
Let's make use of our "stay-home" time and learn new language together!! 

Participants pledget 10 DAI to participate.

## Category

- Learning

## Duration and frequency

- 1 month & everyday 

## What would be the check in criteria?

Each participant has to post screenshot of [Duolingo App](https://www.duolingo.com/) on telegram group. If no screenshot post during the 24 hr period (resets UTC 00:00, then gets 1 strike. 3 strikes and out of the game 

More specification : 
- Language you are Beginner or Intermediate only. (Native or business level is not accepted.)
- Already user of Dulingo app is welcomed to join, but he (she) has to reset current streak and to begin with fresh Day 1 on the first day of challenge. 
- Daily goal has to be set ~~"Serious"~~ "Regular" level (**20XP** per day)
- Members in the same cohort has to be friends on Duolingo app. 
- For mobile users, excuse that you lost 5 hearts and cannot complete daily goal is NOT accepted. 

Please let me know if you have a better ideas or any questions on this.

**Comment by rkalis** [comment]
Why can't someone keep their current streak? It will still be possible to prove that you kept up the streak every day even if it is already running pre-challenge.

**Comment by hanzoh1** [comment]
You are right, I thought it will be easy to track the record from 1 to 30 if it was refreshed new. However, if resetting streak is too much obstacles to participate in this challenge, we can allow users to continue current stroke. thank you for your feedback.

**Comment by makoto** [comment]
Just tried Korean. Not too bad.

**Comment by makoto** [comment]
@hanzoh1 can you paste example proof photo? In my profile page, doesn't really say daily progress https://www.duolingo.com/profile/makoto_inoue

**Comment by crisgarner** [comment]
:+1:

**Comment by hanzoh1** [comment]
@makoto 
For Mobile 
![IMG_7011](https://user-images.githubusercontent.com/42790758/80346559-21651a00-8895-11ea-8130-556f8080b4b2.PNG)
For Desktop 
<img width="399" alt="Screen Shot 2020-04-27 at 14 39 56" src="https://user-images.githubusercontent.com/42790758/80346577-275afb00-8895-11ea-896c-8f1ca9eba421.png">

we can check everyday streak counting from 1 to 30.

**Comment by hanzoh1** [comment]
I changed the daily goal from serious to regular because level serious is too time consuming and challenging.

**Comment by think-in-universe** [comment]
awesome. I'm interested to learn Japanese via the challenge.

**Comment by sazeleo** [comment]
I already have a 15 days streak of learning Spanish. I would love for this event to start. Where I can challenge others. Everyone, can choose their language. They just need to post a screenshot of Duolingo.
@makoto your thoughts?

**Comment by Typto** [comment]
How about adding an interactive session? two people in different languages talk (zoom) for 10 minutes each day to teach each other. the coolest thing is that they can't speak each other's language at all😂
